core/templates/pages/error-pages/error-page.mainpage.html

```diff
@@ -14,4 +14,7 @@
     </div>
     @load('pages/footer_js_libs.html')
   </body>
+  <script type="text/javascript">
+    console.error(`{{error_stack}}`)
```

Comment:
This is jinja, right?
I think we're moving away from using jinja and to use a different mechanism for loading the pages
cc @vojtechjelinek